dashboard/app/models/concerns/user/age.rb

```diff
@@ -0,0 +1,54 @@
+module User::Age
+  extend ActiveSupport::Concern
+
+  AGE_DROPDOWN_OPTIONS = (4..20).to_a << "21+"
```

Comment:
Because this concern is being included in user.rb, all existing calls to `User::AGE_DROPDOWN_OPTIONS` still work.